_posts/2015-02-16-microservices.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: post
+title: Microservices FTW
+author: arto
+excerpt: How to boot up and get some feedback fast
+---
+
+My name is Arto Santala. I've worked in many roles, including software developer, engineer, project manager, scrum master, and architect, as well as a trainer/coach - in companies like Tieturi, ICL, Finnair, and many others. While I've been involved in producing software since the mid-90's, and even before that, I came as new employee in Solita early this year, so it's an exciting new step for me.
+
+Whenever there's a new employee in Software Development department, they will need/get to do an exercise in coding, no matter what they will end up doing here. Idea is to pass on knowledge and ideas both ways, on tools, technologies and architecture, and I think it's a great idea. Equally important is to get to know people, habits, and ways of working together - these also go both ways, since everybody bring along unique set of skills and experiences. So I wanted to write a little blog post on my experiences.
```

Comment:
Eka lause on vähän väljä, voisko tiivistää?  esim.  A new SW developer at Solita first gets to do an ...

The idea is to pass

everybody brings along a unique